src/views/datatable110.blade.php

```diff
@@ -4,12 +4,12 @@
         oTable = jQuery('#{{ $id }}').DataTable({
             "processing": true,
             "serverSide": true,
-            "ajax": "/",
+            "ajax": "{{ isset($options['route']) ? $options['route'] : '/' }}",
```

Comment:
I agree but since option was there I thought it might be the quickest way to solve it at least in the context of the default template.